File: TransCoder_tokenized_test_set_functions/java/SUM_SQUARES_BINOMIAL_COEFFICIENTS.java
static int sumofsquare ( int n ) {
  int [ ] [ ] C = new int [ n + 1 ] [ n + 1 ];
  int i, j;
  for ( i = 0;
  i <= n;
  i ++ ) {
    for ( j = 0;
    j <= min ( i, n );
    j ++ ) {
      if ( j == 0 || j == i ) C [ i ] [ j ] = 1;
      else C [ i ] [ j ] = C [ i - 1 ] [ j - 1 ] + C [ i - 1 ] [ j ];
    }
  }
  int sum = 0;
  for ( i = 0;
  i <= n;
  i ++ ) sum += ( C [ n ] [ i ] * C [ n ] [ i ] );
  return sum;
}

File: results/py/SUM_SQUARES_BINOMIAL_COEFFICIENTS.cpp
int f_filled ( int n ) {
  int C [ n + 1 ] [ n + 1 ] ;
  for ( int j = 0 ;
  j <= n ;
  j ++ ) {
    for ( int i = 0 ;
    i <= n ;
    i ++ ) {
      if ( ( j == 0 || j == i ) && ( C [ i ] [ j ] == 0 ) ) C [ i ] [ j ] = 1 ;
      else C [ i ] [ j ] = ( C [ i - 1 ] [ j - 1 ] + C [ i - 1 ] [ j ] ) ;
    }
  }
  int sum = 0 ;
  for ( int i = 0 ;
  i <= n ;
  i ++ ) sum = sum + ( C [ n ] [ i ] * C [ n ] [ i ] ) ;
  return sum ;
}
